RELEASE.md

```diff
@@ -9,6 +9,7 @@
   - [ ] [Leaflet NPM package page](https://www.npmjs.com/package/leaflet)
   - [ ] files on [Leaflet unpkg page](https://unpkg.com/leaflet@latest/)
 - [ ] Make a new release on [Leaflet's GitHub release page](https://github.com/Leaflet/Leaflet/releases/) with the most important parts of the changelog
+- [ ] Create a zipfile containing all the files inside the `dist` directory, and upload it as an "asset" of the GitHub release
```

Comment:
This ZIP archive is already [uploaded by the CI](https://github.com/Leaflet/Leaflet/blob/81b140bb2668dd6b605ffba899a7a560484d6cf2/.github/workflows/main.yml#L147) at this point, so there is no need to do this manually. It's available from the URL https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/vX.X.X/leaflet.zip

Where `X.X.X` is the version number.